components/asana/completed-task.js

```diff
@@ -0,0 +1,78 @@
+const asana = require("https://github.com/PipedreamHQ/pipedream/components/asana/asana.app.js");
+const get = require("lodash.get");
+
+module.exports = {
+  name: "Task Completed",
```

Comment:
Can you change this to `Task Completed (Instant)` for this and other components that utilize webhooks? We're using this pattern now in the title to highlight the fact that the trigger should work (close to) instantly.